api/pyproject.toml

```
@@ -236,7 +236,8 @@ alibabacloud_gpdb20160503 = "~3.8.0"
 alibabacloud_tea_openapi = "~0.3.9"
 chromadb = "0.5.1"
 clickhouse-connect = "~0.7.16"
-elasticsearch = "8.14.0"
+couchbase = "^4.3.0"
```

```suggestion
couchbase = "~4.3.0"
```
The `^` operator is too wide and forbidden.